### PR TITLE
How the Spryker Code Upgrader works

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-4579

### CODE REVIEW

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ https://api.release.spryker.com/release-group/XXXX
 
 To continue running the Upgrader tool, install the major version manually, and re-run the Upgrader tool.
 
+{% info_block infoBox "Composer dependency conflict" %}
+
+Spryker code upgrader uses the [composer](https://getcomposer.org/) for updating modules and libraries. 
+If the composer detects a conflict, it stops the update process and returns the error message. To resolve the conflict, you need to manually update the conflicting module and re-run the Upgrader pipline.
```
Comment (DmytroKlymanSpryker) [inline docs/scu/dev/how-the-spryker-code-upgrader-works.md:48]: pipline -> pipeline

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md [hunk 2]
```diff
@@ -42,6 +42,13 @@ https://api.release.spryker.com/release-group/XXXX
 
 To continue running the Upgrader tool, install the major version manually, and re-run the Upgrader tool.
 
+{% info_block infoBox "Composer dependency conflict" %}
+
+Spryker code upgrader uses the [composer](https://getcomposer.org/) for updating modules and libraries.
```
Comment (DmytroKlymanSpryker) [inline docs/scu/dev/how-the-spryker-code-upgrader-works.md:47]: Spryker code upgrader -> Spryker Code Upgrader

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md (hunk 1 above)
Comment (DmytroKlymanSpryker) [reply docs/scu/dev/how-the-spryker-code-upgrader-works.md:48]: If the composer detects a conflict, it stops the update process and generates an error message. To resolve the conflict, you must manually update the conflicting module and then re-run the Upgrader pipeline.

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md (hunk 2 above)
Comment (VladislavStrelchenko) [reply docs/scu/dev/how-the-spryker-code-upgrader-works.md:47]: ++

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md (hunk 1 above)
Comment (VladislavStrelchenko) [reply docs/scu/dev/how-the-spryker-code-upgrader-works.md:48]: Thanks ++

**Review by DmytroKlymanSpryker — APPROVED** [review]

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md [hunk 3]
```diff
@@ -42,6 +42,13 @@ https://api.release.spryker.com/release-group/XXXX
 
 To continue running the Upgrader tool, install the major version manually, and re-run the Upgrader tool.
 
+{% info_block infoBox "Composer dependency conflict" %}
+
+Spryker Code Upgrader uses the [composer](https://getcomposer.org/) for updating modules and libraries.
+If the composer detects a conflict, it stops the update process and generates an error message. To resolve the conflict, you must manually update the conflicting module and then re-run the Upgrader pipeline.
```
Comment (VadymSachenko) [inline docs/scu/dev/how-the-spryker-code-upgrader-works.md:48]:
```suggestion
If the composer detects a conflict, it stops the update process and generates an error message. To resolve the conflict, you must manually update the conflicting module and rerun the Upgrader pipeline.
```

File: docs/scu/dev/how-the-spryker-code-upgrader-works.md [hunk 4]
```diff
@@ -42,6 +42,13 @@ https://api.release.spryker.com/release-group/XXXX
 
 To continue running the Upgrader tool, install the major version manually, and re-run the Upgrader tool.
```
Comment (VadymSachenko) [inline docs/scu/dev/how-the-spryker-code-upgrader-works.md:43]:
```suggestion
To continue running the Upgrader tool, install the major version manually, and rerun the Upgrader tool.
```

**Review by VadymSachenko — APPROVED** [review]

**Review by DmytroKlymanSpryker — APPROVED** [review]

**Pull request merged**